portals/devportal/src/main/webapp/site/public/locales/en.json

```diff
@@ -151,6 +151,9 @@
   "Apis.Details.Environments.copied": "Copied",
   "Apis.Details.Environments.copy.to.clipboard": "Copy to clipboard",
   "Apis.Details.Environments.default.url": "( Default Version ) ",
+  "Apis.Details.Environments.download.asyncapi": "Async API Specification",
```

Comment:
AsyncAPI must be a single word. Change all other places as well.

Reply:
Changed